tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature

```diff
@@ -102,7 +102,7 @@ Feature: move files
     When the user tries to move file "lorem.txt" into folder "simple-folder (2)" using the webUI
     Then as "Alice" file "simple-folder (2)/lorem.txt" should not exist in the server
 
-  @issue-ocis-873
+  @issue-ocis-873 @issue-7264 @skipOnOCIS
```

Comment:
can you also add a comment here since this is skipped on ocis like other we have done previously?

Reply:
@SagarGi I think we discussed in the developer discussion not adding comments to the feature until it is a bug demonstration.  
If the issue tag describes the reason for skipping the tests scenario and well-known changes like spaces implementation in ocis which bring some obvious differences in core and ocis. Such changes-related comments should be avoided in the feature file.
For detail information please visit this link: https://github.com/owncloud/docs-server/pull/697/files?short_path=12a0f40#diff-12a0f404a5bbbda65e50018a487cd725797106d670f4308b921d2081dc4b3516